k8s/aat/common/rd/judicial-data-load.yaml

```diff
@@ -19,7 +19,7 @@ spec:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/rd/judicial-data-load:prod-dd3f7021
+      image: hmctspublic.azurecr.io/rd/judicial-data-load:latest
```

Comment:
Could you please check if this is the correct syntax? We have asked all the teams not to use **latest**